### DISCUSSION TITLE
Reload configuration does not work when the config is symlinked

### DISCUSSION THREAD

**Discussion by rgomezcasas** [ghostty-org/ghostty#2571] [Q&A] [answered]
Since I symlinked the config in order to back it up in my dotfiles, the reload config action has stopped working:

![Image](https://github.com/user-attachments/assets/c683d12d-05d7-4d56-90f2-66fced0cf9a3)

I have to close and reopen Ghostty in order to reload the config.

I've also tried to symlink only the `config` file instead of the whole folder, but the problem is the same.

My environment:
* macOS 15.1
* Ghostty d22464fb, build 7860

Thanks!

**Comment by mitchellh** [comment, marked as answer]
I can't reproduce this. Are you sure the configuration file exists and is readable? Are you sure you're reloading the configuration? Are you sure the keys you're reloading are reloadable (i.e. font-size does nothing to existing windows)? I tried both the directory and file and they both reloaded properly.

**Comment by rgomezcasas** [comment]
Sorry, it was that. I was changing the font size, but as the config says, it only affects to new windows, not to the current ones.

Thanks and sorry again!
